### PR TITLE
Add user session buttons

### PR DESCRIPTION
**Pull request by Applepie9**

### CODE REVIEW

File: app/views/home/_header.html.erb [hunk 1]
```diff
@@ -1 +1,33 @@
...
+        
+
+		<% if user_signed_in? %>
+
+		  <li class="nav-item">
+		    <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class:"nav-link" %>
+		  </li>
+
+		  <li class="nav-item">
+			  <%= link_to "Edit Profile", edit_user_registration_path, class:"nav-link" %>
```
Comment (kaizencodes) [inline app/views/home/_header.html.erb]: do we have this?

File: app/views/home/_header.html.erb [hunk 2]
```diff
@@ -1 +1,33 @@
...
+  <div class="container">
+    <nav>
+      <ul>
+      	<li class="nav-item">
+          <%= link_to "Home", root_path, class:"nav-link" %>
+     	</li>
+        
+
+		<% if user_signed_in? %>
+
```
Comment (kaizencodes) [inline app/views/home/_header.html.erb]: we should display the user email here.

**Review by kaizencodes — COMMENTED** [review]
indentation seem off in app/views/home/_header.html.erb.
use 2 spaces like in ruby.

File: app/views/home/_header.html.erb (hunk 1 above)
Comment (Applepie9) [reply app/views/home/_header.html.erb]: yes, comes with devise gem

**Review by kaizencodes — APPROVED** [review]

**Comment by Applepie9** [comment]
Add user session buttons

**Pull request merged**